internal/api/create_credentials_v1alpha1.go

```diff
@@ -18,7 +19,7 @@ type credentials struct {
 	name           string
 	credType       string
 	repoURL        string
-	repoURLPattern string
+	repoURLISRegex bool
```

Comment:
Nit:

```suggestion
	repoURLIsRegex bool
```

Reply:
That was a mistake. Thanks.

Will change it locally so I can use ID "rename symbol" functionality.

Reply:
Fixed in https://github.com/akuity/kargo/pull/1712/commits/11744d906112ac41a61f4d801c6d91af4e5cc990